man/man8/zfs-send.8

```diff
@@ -33,9 +32,10 @@
 .Dd April 15, 2021
 .Dt ZFS-SEND 8
 .Os
+.
 .Sh NAME
 .Nm zfs-send
-.Nd Generate a send stream, which may be of a filesystem, and may be incremental from a bookmark.
+.Nd generate back-up stream of ZFS dataset
```

Comment:
maybe: "generate a backup stream of a ZFS dataset"

Reply:
as above